client/src/components/Loading.tsx

```diff
@@ -0,0 +1,25 @@
+import { ScanSpinner } from 'react-craftify-spinners';
+import { IoMdArrowBack as Arrow } from 'react-icons/io';
+
+import Button from './Button';
+
+const Loading = ({ prev }: { prev: () => void }) => {
+  return (
+    <div className='max-w-48 m-auto'>
+      <Button
+        icon={Arrow}
+        variant='tertiary'
+        className='absolute left-16'
+        onClick={() => prev()}
+      >
+        Back
```

Comment:
should we allow the user to go back while the notes is generating?

Reply:
My thinking was that it would be good for them to have a way to cancel it so if they change their mind or realize they wanted to change the customization they wouldn't have to wait for it to finish. I don't think it's working perfectly right now because the state management is overall not that great right now so if they go back it might not actually cancel/delete these notes, but that's the general idea